src/components/profile/Profile.jsx

```diff
@@ -3,15 +3,20 @@ import { useSelector } from 'react-redux';
 import MyRockets from './MyRockets';
 import MyMissions from './MyMissions';
 import styles from './Profile.module.css';
+import {selectAllMissions} from '../../Redux/mission/missionSlice'
+import { useEffect } from 'react';
 
 function Profile() {
+  const { mission } = useSelector(selectAllMissions);
+  const joinMission = mission.filter((x) => x.joined === true);
   const rockets = useSelector((state) => state.rockets).filter(
     (rocket) => rocket.reservation,
   );
```

Comment:
_[OPTIONAL]_ Since the map function is iterating over the missions array, it will be a huge favour to your code reader to represent each object with a more descriptive identifier instead of `x` 

for example `mission.filter(x =>{ your code.....})` use `mission.filter(mission =>{ your code.....})`